### PR TITLE
mrd-1491_update-to-readme-for-maintenance-banner: added instructions …

### PR DESCRIPTION
**Pull request by bills7281**
…to README on how to use maintenance banner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
 * [Deployment / Helm](./docs/helm-deploy.md) - including how to roll back a deployment
 * [Runbook](./RUNBOOK.md)
 
+### Maintenance Banner
+* To display a maintenance banner to make users aware of any planned outages five environment variables are required. These are available for each environment in the helm_deploy folder -
+  * **SHOW_NOTIFICATION** - this must be set to true for the banner to manifest, by default it is false
+  * **NOTIFICATION_HEADER** - the header of the banner
+  * **NOTIFICATION_BODY** - the text in the body of the banner
+  * **START_DATE** - must be written in ISO format eg/YYYY-MM-DD, the banner will only be displayed **_after_** this date
```
Comment (DarrenW-HMPPS) [inline README.md:34]: When this says "after" this date, does it mean that if I set it to 3 Sept, the first time the notification banner will be displayed is 4th September?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
 * [Deployment / Helm](./docs/helm-deploy.md) - including how to roll back a deployment
 * [Runbook](./RUNBOOK.md)
 
+### Maintenance Banner
+* To display a maintenance banner to make users aware of any planned outages five environment variables are required. These are available for each environment in the helm_deploy folder -
+  * **SHOW_NOTIFICATION** - this must be set to true for the banner to manifest, by default it is false
```
Comment (DarrenW-HMPPS) [inline README.md:31]: How does this interact with the start date and end date?

File: README.md (hunk 1 above)
Comment (bills7281) [reply README.md:34]: It's inclusive, I'll update and add a link to the GDS document Beck signposted me to

File: README.md (hunk 2 above)
Comment (bills7281) [reply README.md:31]: This is a sort of master switch. If SHOW_NOTIFICATION is false (default), the banner won't be displayed regardless of start/end dates. Andrew's old 'Assess risks and needs' project follows the same pattern.

Comment (DarrenW-HMPPS) [inline README.md:31]: SHOW_NOTIFICATION needs removing from this ReadMe (and the text "five environment variables are required." should be changed to say four.

File: README.md [hunk 3]
```diff
@@ -26,6 +26,15 @@ Read more on the [Confluence space](https://dsdmoj.atlassian.net/wiki/spaces/MRD
...
+  * **NOTIFICATION_HEADER** - the header of the banner
+  * **NOTIFICATION_BODY** - the text in the body of the banner. This must conform to GDS standards [see link](https://design-system.service.gov.uk/components/notification-banner/)
+  * **START_DATE** - must be written in ISO format eg/YYYY-MM-DD, the banner will only be displayed **_on or after_** this date
+  * **END_DATE** - must be written in ISO format eg/YYYY-MM-DD, the banner will only be displayed **_on or before_** this date
+
+
```
Comment (DarrenW-HMPPS) [inline README.md:37]: I think there are also instances of SHOW_NOTIFICATION that appear in env files etc. that should also be removed.

File: README.md (hunk 2 above)
Comment (bills7281) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (bills7281) [reply README.md:36]: Done

**Review by DarrenW-HMPPS — APPROVED** [review]

**Pull request merged**